PNA.mdk

```diff
@@ -860,6 +860,24 @@ extern definitions being moved from the PSA specification into a
 separate standard library of P4 extern definitions, and if this is
 done, both the PSA and PNA specifications will reference that.
 
+## Extern Objects for Inline Accelerators {#sec-extern-restrictions}
+
+A variety of inline accelerators can be present on a PNA target. These
+accelerators perform specific functions on a packet. These functions are
```

Comment:
Suggest reminding that the accelerators execute after the pipeline, e.g.:
> These accelerators perform specific functions on a packet, _usually after the `Deparser` has finished executing_.

Reply:
added